src/nettests/index.js

```diff
@@ -1,9 +1,106 @@
+import camelCase from 'camelcase'
```

Comment:
w-h-a-t--i-s--t-h-a-t?! :-D

Reply:
(lol, below I've seen how it works, and I really like how in node there is a small library for everything; I find this to be very Unix like and in general to be a sensible approach)